samples/apps/autogen-studio/autogenstudio/workflowmanager.py

```diff
@@ -942,9 +1050,27 @@ async def a_receive(
         silent: Optional[bool] = False,
     ) -> None:
         if self.a_message_processor:
-            await self.a_message_processor(sender, self, message, request_reply, silent, sender_type="agent")
+            if isinstance(message, dict) and message.get("tool_calls"):
+                tool_calls = [
+                    {
+                        "name": func["name"],
+                        #"arguments": ', '.join(f'{key}: {value}' for key, value in json.loads(func["arguments"]).items())
```

Comment:
Note the comment above, which is why arguments are not displayed in the async version.